src/components/Hero.style.js

```diff
@@ -0,0 +1,75 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const DivStyled = styled.div`
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+`;
+
+export const H1Styled = styled.h1`
+  font-family: Bitter;
+  font-size: 38px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  text-align: center;
+  color: var(--gray-dark);
```

Comment:
did you define `gray-dark` somewhere? Otherwise the colors should be moved to a [theme](https://styled-components.com/docs/advanced#theming)